tests/Tests/100__deploy/130__operators/130__rhods_operator/132__traefik-proxy_restart_verification.robot

```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+Documentation       132 - RHODS_TRAEFIK_PROXY_CONTAINER_RESTART_VERIFICATION
+...                 Verify that rhods traefik-proxy container
+...                 restart verification
+...
+...                 = Variables =
+...                 | Namespace         | Required |    RHODS Namespace/Project for RHODS operator POD |
+...                 | LABEL_SELECTOR    | Required |    Label selector for traefik proxy|
+
+Resource            ../../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+
+
+*** Variables ***
+${NAMESPACE}            redhat-ods-applications
+${LABEL_SELECTOR}       name=traefik-proxy
+
+
+*** Test Cases ***
+Verify Traefik Proxy Containers Restart
+    [Documentation]    Verify traefik proxy
+    ...    container restart
+    [Tags]    Sanity
+    ...       ODS-1163    KnownIssues
+    ${p_names}    Get POD Names    ${NAMESPACE}    ${LABEL_SELECTOR}
+    Verify Restart Container Verification    ${p_names}
+
+
+*** Keywords ***
+Verify Restart Container Verification
+    [Documentation]    Get and verify container restart
+    ...    Counts for pods
+    [Arguments]    ${names}
+    # Todo:We should move this keyword to common folder in future
```

Comment:
why not moving it now?

Reply:
I want to see if people really need this if i see it is needed by another script will immediately move at that time

Reply:
hm I see this as a common keyword. Maybe today it is not needed from others, but there might be the case in future.

Reply:
hmm..i know that's the reason i put it in todo ..For now i will keep it like this

Reply:
by keeping it there anyone may re-create it because not finding in the Pod resource file..then if the reviewer remember about this keyword will tell to delete the new one and move the old one to resource etc. I don't see much value to leave the keyword now. That's my opinion. let's hear also from others

Reply:
i will move that no worries